xla/service/cpu/backend_config.proto

```diff
@@ -8,14 +8,50 @@ message BackendConfig {
   // outer-most dimension first). Used by the parallel cpu backend to partition
   // HLOs into parallel tasks.
   repeated int64 outer_dimension_partitions = 1;
-  // Configuration to be used by oneDNN matmul
-  OneDnnMatMulConfig onednn_matmul_config = 2;
-  OneDnnLayerNormConfig onednn_layer_norm_config = 3;
+  oneof backend_config_oneof {
+    // Configuration to be used by oneDNN matmul
+    OneDnnMatMulConfig onednn_matmul_config = 2;
+    // Configuration to be used by oneDNN layer norm
+    OneDnnNormConfig onednn_layer_norm_config = 3;
+    // Configuration to be used by oneDNN convolution
+    OneDnnConvolutionConfig onednn_conv_config = 4;
+  }
```

Comment:
Note that we usually can't delete fields in protobuf (can only add new fields and, at most, mark old fields as deprecated -- to maintain forward/backward compatibility). But since these protos are only used in internal XLA C++ files, it should be okay to drop the compatibility here.